src/api/routes/v1/index.ts

```diff
@@ -18,6 +18,7 @@ import handleValidatorReport from './validator-report'
 const api = createRouter()
 
 api.use('/health', handleHealth)
+api.use('/metrics/:network', handleWebSocketHealthMetrics)
```

Comment:
Can you add documentation for this API endpoint inside the `ARCHITECTURE.md` file and `src/api/routes/v1/info.ts` file?